doc/api/net.md

```diff
@@ -205,34 +308,37 @@ persist*, they are removed when the last reference to them is closed. Do not
 forget JavaScript string escaping requires paths to be specified with
 double-backslashes, such as:
 
-    net.createServer().listen(
-        path.join('\\\\?\\pipe', process.cwd(), 'myctl'))
+```js
+net.createServer().listen(
+  path.join('\\\\?\\pipe', process.cwd(), 'myctl'))
+```
 
-The parameter `backlog` behaves the same as in
-[`server.listen(port[, hostname][, backlog][, callback])`][`server.listen(port, host, backlog, callback)`].
+The `backlog` argument is the maximum size of the queue of pending connections.
+The maximum possible value will be determined by the operating system.
 
 ### server.listen(port[, hostname][, backlog][, callback])
 <!-- YAML
 added: v0.1.90
 -->
 
-Begin accepting connections on the specified `port` and `hostname`. If the
-`hostname` is omitted, the server will accept connections on any IPv6 address
-(`::`) when IPv6 is available, or any IPv4 address (`0.0.0.0`) otherwise. Use a
-port value of `0` to have the operating system assign an available port.
+The `server.listen()` method instructs the server to begin accepting connections
```

Comment:
here as well it should reference/link to the options-based variant

duplicating the text is not only a problem from a maintainability point of view, its also a problem from a readers point of view.

ATM, without reading the code or doing a side-by-side and line-by-line textual comparison of the options-based and port+host based API documentation, I cannot know that the handling of (for example) hostname defaults is identical for the two APIs (because they are in fact the same API). The docs should make clear that they are alternative ways of providing the options, but the behaviour associated with the options is identical, no matter how they are passed (positional arg or options object).